pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "chessticulate-api"
-version = "0.3.10"
+version = "0.3.11"
```

Comment:
since we've expanding on the API rather than fixing some small thing, I'd put this to 0.4.0